sdks/wasm/framework/src/WebAssembly.Net.Http/WasmHttpMessageHandler.cs

```diff
@@ -2,11 +2,14 @@
 using System.IO;
 using System.Net;
 using System.Net.Http;
+using System.Runtime.CompilerServices;
 using System.Threading;
 using System.Threading.Tasks;
 using WebAssembly.Core;
 using WebAssembly.Host;
 
+[assembly: InternalsVisibleTo ("System.Net.Http")]
+
```

Comment:
Why is this not strong-named?

We are using this in `System.dll`:
https://github.com/mono/mono/blob/e6e9d5f6eb5488d9f87ab39727d76fb2ca81702c/mcs/class/System/Assembly/AssemblyInfo.cs#L71

Reply:
Addressed in recent commits